pkg/controller/jobs/job/job_controller_test.go

```diff
@@ -312,7 +311,7 @@ func TestEquivalentToWorkload(t *testing.T) {
 		},
 		"partial admission different count (unsuspended)": {
 			job: func() *Job {
-				j := &Job{baseJobPartialAdmission.DeepCopy()}
+				j := fromObject(baseJobPartialAdmission.Object().DeepCopyObject())
```

Comment:
I think this should work

```suggestion
				j := fromObject(baseJobPartialAdmission.DeepCopy())
```

Reply:
done